src/send/InviteRewardsCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { notificationInvite } from 'src/images/Images'
+import { inviteRewardsTypeSelector } from 'src/send/selectors'
+import { InviteRewardsType } from 'src/send/types'
+import Colors from 'src/styles/colors'
+import { Spacing } from 'src/styles/styles'
+
+export default function InviteRewardsCard() {
+  const { t } = useTranslation()
+  const inviteRewardsType = useSelector(inviteRewardsTypeSelector)
+
+  // Default to NFT invite rewards banner
```

Comment:
this doesn't seem safe. we add invite rewards features relatively often and it would be easy to forget to update this. how about logging an error and returning null if the type of invite rewards isn't nft or cusd? you can also add static validation with something like this:
```
const inviteRewardsToTranslationPrefix: Record<InviteRewardsType, string> = {
[InviteRewardsType.CUSD]: 'inviteRewardsBannerCUSD',
[InviteRewardsType.NFT]: 'inviteRewardsBanner',
}
const inviteRewardsTranslationPrefix = inviteRewardsToTranslationPrefix[inviteRewardsType]
const titleKey = `${inviteRewardsPrefix}.title`
const bodyKey = ...
```

Reply:
Fixed in 2425b09f570a00993c0664900786b1623e61a8d4!